python/sparkdl/estimators/keras_image_file_estimator.py

```diff
@@ -264,7 +265,7 @@ def fitMultiple(self, dataset, paramMaps):
                      existence of a sufficiently large (and writable) file system, users are
                      advised to not train too many models in a single Spark job.
         """
-        [self._validateParams(pm) for pm in paramMaps]
+        assert all([self._validateParams(pm) for pm in paramMaps])
```

Comment:
Q: why the assert? looks like we already raise a ValueError if a paramMap is invalid in `_validateParams`

Reply:
yeah I agree, you could move the assert above to signal that it is always expected to be there

Reply:
`assert any([...])` was changed to `_ = [...]` for readability, because it suggested that the error would be thrown because of the assert, but in reality the error was thrown in `_validateParams`